tests/test_edx_east_roles.sh

```diff
@@ -38,6 +38,10 @@ ansible-playbook -i localhost, -c local --tags edxapp_cfg edxapp.yml -e edxapp_u
     -e@languages.yml \
     -e@server-vars.yml \
     -e EDXAPP_PREVIEW_SITE_NAME=""
+    -e EDXAPP_SESSION_INACTIVITY_TIMEOUT_IN_SECONDS=""
+    -e EDXAPP_ALL_LANGUAGES_OVERRIDE=""
```

Comment:
FYI the language override might not work because we're treating it like a string here when it's actually a dictionary. don't change it until we are sure that's a problem.